include/simdutf/implementation.h

```diff
@@ -276,6 +489,25 @@ simdutf_warn_unused size_t convert_latin1_to_utf8(const char *input,
 simdutf_warn_unused size_t
 convert_latin1_to_utf8_safe(const char *input, size_t length, char *utf8_output,
                             size_t utf8_len) noexcept;
+#if SIMDUTF_CPLUSPLUS20
+template <typename InputSpan, typename OutputSpan>
+  requires detail::span_of_byte_like<InputSpan> &&
+           detail::output_span_of_byte_like<OutputSpan>
+simdutf_really_inline simdutf_warn_unused size_t convert_latin1_to_utf8_safe(
+    const InputSpan &input, OutputSpan &&utf8_output) noexcept {
```

Comment:
Here you are using camel case (InputSpan). I personnally do not mind if it is deliberate (for some reason).